packages/plugins/quorum/README.md

```diff
@@ -0,0 +1,346 @@
+```
+deploying contracts
+SimpleStorage already deployed at 0x9d13C6D3aFE1721BEef56B55D303B09E021E27ab
+finished deploying contracts
+```
+Now, do the same thing as before and retrieve the value stored in the contract:
+```
+Embark (nodeone) > SimpleStorage.get()
+# 100
+```
+Again, as expected, we get the value of 100, matching the constructor arguments. This is expected because Node 1 deployed the `SimpleStorage` contract privately between nodes 1 and 7.
+
+Let's take this one step further and check the value stored in `SimpleStorage` on a node that was not included in the private contract deploy.
+
+Exit the current Embark instance with `ctrl+c`, then start up a new instance in the `nodetwo` environment:
+```
+<ctrl+c>
+embark console nodetwo
+```
+Notice the warning that the bytecode check is skipped, and also that Embark knows the contract has already been deployed, **even though the bytecode doesn't exist on Node 2**:
+```
+compiling solidity contracts...
+deploying contracts
+WARNING: Skipping bytecode check for SimpleStorage deployment. Performing an embark reset may cause the contract to be re-deployed to the current node regardless if it was already deployed on another node in the network.
+SimpleStorage already deployed at 0x9d13C6D3aFE1721BEef56B55D303B09E021E27ab
+finished deploying contracts
+```
+Again, let's see now what Node 2 has stored for the `SimpleStorage` contract value:
+```
+Embark (nodetwo) > SimpleStorage.get()
+# 0
+```
+The result is `0`, which is exepcted because Node 2 does not contain the bytecode for `SimpleStorage` and does not have its values stored in its Ethereum state.
+
+## 6. Sending a private transaction
+Let's see how the `embark-quorum` plugin allows us to send private transactions.
+
+First, run embark console in the `nodeone` environment to see what the current value of the `SimpleStorage` contract is.
+
+```
+# /bin/bash
+
+embark console nodeone
+```
+Once started, get our contract value in the console prompt:
+```
+Embark (nodeone) > SimpleStorage.get()
+# 100
+```
+Now, let's update this value only for Node 7. In the console, we're going to send a private transaction between Node 1 and Node 7. This transaction is simply going to set the `SimpleStorage` value to `333` by calling the `SimpleStorage.set(333)` method in a transaction. Run each of these lines individually in the console:
+```
+Embark (nodeone) > from = web3.eth.defaultAccount
+Embark (nodeone) > privateFor = ['ROAZBWtSacxXQrOe3FGAqJDyJjFePR5ce4TSIzmJ0Bc=']
+Embark (nodeone) > await SimpleStorage.methods.set(333).send({ from, privateFor })
+```
+We can check that the transaction set our `SimpleStorage` value to `333`:
+```
+Embark (nodeone) > SimpleStorage.get()
+# 333
+```
+Now, let's check Node 7:
+```
+<ctrl+c>
+embark console nodeseven
+```
+Then, in the console prompt:
+```
+Embark (nodeseven) > SimpleStorage.get()
+# 333
+```
+Great! We have privately set our `SimpleStorage` contract value only in Node 7.
+
+---
+
```

Comment:
Great docs. Since this is pretty much a guide/tutorial, maybe you should add a conclusion. Something like, now that you know the basics, you can use Quorum in your own Dapp or start a new one. Also remind them that they don't have to use all seven nodes to something like that.

Reply:
Added, thanks!